src/mongo-service.js

```diff
@@ -161,17 +160,20 @@ class MongoService extends MongoQueryService {
     entity = await updateFn(entity);
     const updated = await this._validate(entity);
 
-    await this._collection.update(
+    const updateCallback = () => {
+      this._bus.emit('updated', {
+        doc: updated,
+        prevDoc: doc,
+      });
+    };
+
+    await this._collection.updateOne(
       { ...query, _id: doc._id },
-      updated,
-      { ...options, replaceOne: true },
+      { $set: updated },
```

Comment:
I think it should be replace.

https://github.com/paralect/node-mongo/pull/47/files

Reply:
instead of updateOne? this one? https://docs.mongodb.com/manual/reference/method/db.collection.replaceOne/
bacause updateOne with `replaceOne: true` was monk's feature